tests/test_mixed_simulator_device.py

```diff
@@ -657,56 +657,23 @@ def test_expval_two_wires_with_parameters(
         assert np.isclose(res, expected_output, **tol)
 
     @pytest.mark.parametrize("use_super", [False, True])
-    def test_four_qubit_random_circuit(self, shots, use_super, mocker):
-        """Test a four-qubit random circuit with the whole set of possible gates,
-        the test is analog to a failing device test and is used to check the try/except
+    def test_mixed_sim_expval(self, shots, use_super, mocker):
```

Comment:
```suggestion
    def test_mixed_sim_expval(self, use_super, mocker):
```

I think we can just remove this since it is not used in the test.

Reply:
Yeah I thought you could too but it's mandatory (I think some fixture is providing it to all of the tests in the test class). 🤣

Reply:
Oh oops, right... Ok that's fine then 🤣

Reply:
True ... I didn't even bother to look 🤦🏼 . All of them are like that.

Reply:
It is completely useless.